examples/deployments/render-terraform/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "chroma_core_repo_url" {
+    description = "The URL of the chroma-core repository"
+    type        = string
+    default     = "https://github.com/chroma-core/chroma"
+}
+
+variable "chroma_release" {
+  description = "The chroma release to deploy"
+  type        = string
+  default     = "0.4.13"
+}
+
+variable "region" {
+  type    = string
+  default = "oregon"
+}
+
+variable "render_plan" {
+  default     = "starter"
+  description = "The Render plan to use. This determines the size of the machine. NOTE: Terraform Render provider uses Render's API which requires at least starter plan."
+  type = string
+}
+
+variable "enable_auth" {
```

Comment:
same nit here: looks like this is captured in an env var and not used in the Terraform, let's remove

Reply:
Made use of it :)

Reply:
We can now enable/disable auth as with other blueprints.